_episodes/03-create.md

```diff
@@ -123,6 +123,16 @@ then run a text editor called Nano to create a file called `draft.txt`:
 ~~~
 $ cd thesis
 $ nano draft.txt
+$ ls -F
+~~~
+{: .bash}
+
+Apparently, if we create empty file using nano, the file won't be saved.
+But we can use command `touch` to create a file called `draft.txt`:
+
+~~~
+$ touch draft.txt
+$ ls -F
```

Comment:
I'd be happy to see `touch` in the lecture, as per issue #585 , I believe it needs some more words though.
I'm not sure I understand what you mean by "apparently if we create empty file using nano the file won't be saved"?
If you go through the usual write out>file name>exit the empty file is indeed created, definitely more time consuming and more complicated than needed IMO, but the file is saved.

Reply:
Yes, you are right! I didn't explore it enough, the file can't be save by using "cntrl + x", but do with "cntrl +o" as you pointed out.

I still think nano is kinda of challenge (most people don't how to get out for their fist time), if you this material is focus on learners with zero background CML interface. 

Thanks!